hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/BinaryCompatibilityNullFileGenerator.java

```diff
@@ -1,352 +1,7 @@
-import com.hazelcast.client.impl.protocol.codec.TransactionalMapGetCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMapGetForUpdateCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMapIsEmptyCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMapKeySetCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMapKeySetWithPredicateCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMapPutCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMapPutIfAbsentCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMapRemoveCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMapRemoveIfSameCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMapReplaceCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMapReplaceIfSameCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMapSetCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMapSizeCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMapValuesCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMapValuesWithPredicateCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMultiMapGetCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMultiMapPutCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMultiMapRemoveCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMultiMapRemoveEntryCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMultiMapSizeCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalMultiMapValueCountCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalQueueOfferCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalQueuePeekCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalQueuePollCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalQueueSizeCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalQueueTakeCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalSetAddCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalSetRemoveCodec;
-import com.hazelcast.client.impl.protocol.codec.TransactionalSetSizeCodec;
-import com.hazelcast.client.impl.protocol.codec.XATransactionClearRemoteCodec;
-import com.hazelcast.client.impl.protocol.codec.XATransactionCollectTransactionsCodec;
-import com.hazelcast.client.impl.protocol.codec.XATransactionCommitCodec;
-import com.hazelcast.client.impl.protocol.codec.XATransactionCreateCodec;
-import com.hazelcast.client.impl.protocol.codec.XATransactionFinalizeCodec;
-import com.hazelcast.client.impl.protocol.codec.XATransactionPrepareCodec;
-import com.hazelcast.client.impl.protocol.codec.XATransactionRollbackCodec;
+import com.hazelcast.client.impl.protocol.codec.*;
```

Comment:
This file is generated. Hence, it is better to change its .template file at the code-generator repo and then auto-generate it. Otherwise, it will be overriden.

Similarly, all compatibility tests are also auto generated. Did you regenerate or manually change them? Better to change the code-generator first.